app/src/main/res/navigation/authentication_navigation.xml

```diff
@@ -7,7 +7,7 @@
     <fragment
         android:id="@+id/signUpFragment"
         android:name="com.example.learingrealmandretrofit.fragment.SignUpFragment"
-        android:label="SignUpFragment"
+        android:label="SIGN UP"
```

Comment:
Where this label should be?

Reply:
In toolbar.
![photo_2021-11-27_20-43-51](https://user-images.githubusercontent.com/78955249/143718760-812f9075-ba9b-4bfc-87aa-4bede5a96685.jpg)

Reply:
It should be in strings file 🤦